src/arrayMethodSplice.js

```diff
@@ -4,8 +4,61 @@
  * Implement method Splice
  */
 function applyCustomSplice() {
-  [].__proto__.splice2 = function(start, deleteCount, ...items) {
+  [].__proto__.splice2 = function(
+    startIndex,
+    deleteCount = startIndex > 0 ? (this.length - startIndex) : -startIndex,
```

Comment:
will be better to set initial values depending on some check inside the function, not in the arguments